Library/DunningCampaign.cs

```diff
@@ -0,0 +1,189 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Xml;
+
+namespace Recurly
+{
+
+  /// <summary>
+  /// A Dunning Campaign in Recurly.
+  /// </summary>
+  public class DunningCampaign : RecurlyEntity
+  {
+    /// <summary>
+    /// Unique ID assigned to this dunning campaign.
+    /// </summary>
+    public string Id { get; private set; }
+
+    public string Name { get; private set; }
+
+    public string Code { get; private set; }
+
+    public string Description { get; private set; }
+
+    public bool DefaultCampaign { get; private set; }
+
+    private List<DunningCycle> _dunningCycles;
+
+    public List<DunningCycle> DunningCycles
+    {
+        get { return _dunningCycles ?? (_dunningCycles = new List<DunningCycle>()); }
+        set { _dunningCycles = value; }
+    }
+
+    public DateTime? CreatedAt { get; private set; }
+
+    public DateTime? UpdatedAt { get; private set; }
+
+    public DateTime? DeletedAt { get; private set; }
+
+    internal const string UrlPrefix = "/dunning_campaigns/";
+
+    public List<string> PlanCodes { get; private set;}
```

Comment:
I think that we want to make this private as there is no reason to expose the availability of this variable to the developer. It is used solely as a storage location between `BulkUpdate` being called and the `WriteBulkUpdateXML` method being executed.

```suggestion
    private List<string> PlanCodes { get; private set;}
```

Reply:
yep good call - fixed